x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/coverage_overview/coverage_overview.cy.ts

```diff
@@ -0,0 +1,170 @@
+      .contains('Disabled custom rule')
+      .should('not.exist');
+    cy.get(COVERAGE_OVERVIEW_ENABLE_ALL_DISABLED_BUTTON).should('not.be.disabled');
+
+    // only filtering for prebuilt rules
+    selectCoverageOverviewActivityFilterOption('Enabled rules');
+    openTechniquePanel(getMockThreatData().technique.name);
+    cy.get(COVERAGE_OVERVIEW_POPOVER_ENABLED_RULES).contains('Enabled prebuilt rule');
+    cy.get(COVERAGE_OVERVIEW_POPOVER_ENABLED_RULES)
+      .contains('Enabled custom rule')
+      .should('not.exist');
+    cy.get(COVERAGE_OVERVIEW_POPOVER_DISABLED_RULES).contains('Disabled prebuilt rule');
+    cy.get(COVERAGE_OVERVIEW_POPOVER_DISABLED_RULES)
+      .contains('Disabled custom rule')
+      .should('not.exist');
+    cy.get(COVERAGE_OVERVIEW_ENABLE_ALL_DISABLED_BUTTON).should('not.be.disabled');
+
+    // only filtering for custom rules
+    selectCoverageOverviewSourceFilterOption('Custom rules');
+    selectCoverageOverviewSourceFilterOption('Elastic rules'); // Turn off filter
+
+    openTechniquePanel(getMockThreatData().technique.name);
+    cy.get(COVERAGE_OVERVIEW_POPOVER_ENABLED_RULES)
+      .contains('Enabled prebuilt rule')
+      .should('not.exist');
+    cy.get(COVERAGE_OVERVIEW_POPOVER_ENABLED_RULES).contains('Enabled custom rule');
+    cy.get(COVERAGE_OVERVIEW_POPOVER_DISABLED_RULES)
+      .contains('Disabled prebuilt rule')
+      .should('not.exist');
+    cy.get(COVERAGE_OVERVIEW_POPOVER_DISABLED_RULES).contains('Disabled custom rule');
+    cy.get(COVERAGE_OVERVIEW_ENABLE_ALL_DISABLED_BUTTON).should('not.be.disabled');
+
+    // filtering for search term
+    selectCoverageOverviewSourceFilterOption('Elastic rules');
+    filterCoverageOverviewBySearchBar('Enabled custom rule');
+
+    openTechniquePanel(getMockThreatData().technique.name);
+    cy.get(COVERAGE_OVERVIEW_POPOVER_ENABLED_RULES)
+      .contains('Enabled prebuilt rule')
+      .should('not.exist');
+    cy.get(COVERAGE_OVERVIEW_POPOVER_ENABLED_RULES).contains('Enabled custom rule');
+    cy.get(COVERAGE_OVERVIEW_POPOVER_DISABLED_RULES)
+      .contains('Disabled prebuilt rule')
+      .should('not.exist');
+    cy.get(COVERAGE_OVERVIEW_POPOVER_DISABLED_RULES)
+      .contains('Disabled custom rule')
+      .should('not.exist');
+    cy.get(COVERAGE_OVERVIEW_ENABLE_ALL_DISABLED_BUTTON).should('be.disabled');
+  });
+
+  it('enables all disabled rules', () => {
+    selectCoverageOverviewActivityFilterOption('Disabled rules');
+    openTechniquePanel(getMockThreatData().technique.name);
+    enableAllDisabledRules();
+
+    // Should now render all rules in "enabled" section
+    openTechniquePanel(getMockThreatData().technique.name);
+    cy.get(COVERAGE_OVERVIEW_POPOVER_ENABLED_RULES).contains('Enabled prebuilt rule');
+    cy.get(COVERAGE_OVERVIEW_POPOVER_ENABLED_RULES).contains('Enabled custom rule');
+    cy.get(COVERAGE_OVERVIEW_POPOVER_ENABLED_RULES).contains('Disabled prebuilt rule');
```

Comment:
Not sure I understand this one here: the Enabled Rules popover contains "Disabled prebuilt rule" and "Disabled custom rule"?

Reply:
I named the rules this way to help see what their initial state was when created thinking it'd be more intuitive than test rule 1, 2, 3, etc., when we enable all the rules in this test the rule originally named "Disabled" will become enabled